.github/workflows/submitty_ci.yml

```diff
@@ -253,7 +253,6 @@ jobs:
               sudo pip3 install docker
               sudo pip3 install paramiko
               sudo pip3 install tzlocal
-              sudo pip3 install sqlalchemy
```

Comment:
this was a duplicate install line, removed it for the single install line on 270.